approval_polls/models.py

```diff
@@ -1,52 +1,103 @@
+from django.core.urlresolvers import reverse
 from django.db import models
 from django.contrib.auth.models import User
 
+from django.utils import timezone
+
+
 class Poll(models.Model):
-  question = models.CharField(max_length=200)
-  pub_date = models.DateTimeField('date published')
-  user = models.ForeignKey(User)
+    '''A Poll class. This is the top level container for a Poll.
+
+    referenced by Choices model and the Ballot model
+    '''
+
+    question = models.CharField(max_length=200)
+    pub_date = models.DateTimeField('date published')
+    user = models.ForeignKey(User)
+    open_date = models.DateField()
+    close_date = models.DateField()
+    suspended = models.BooleanField(default=False)
+
+    def is_closed(self):
+        '''Determine if the poll is open or closed
+
+        Arguments:
+            None
+
+        Returns:
+            boolean: True if the close_date < datetime.datetime.now()
+                     otherwise False
+        '''
+
+        now = timezone.now()
+        # If now is greater than the close date the close date is in the past and will return True
+        # If self.close_date is greater than now then close date is in the future and this returns False
+        return now > self.close_date
```

Comment:
I agree that comments are helpful but in this case they are just repeating what the code is doing. I think comments are useful when they talk about why something is being done rather than what is being done because that should be evident from the code.